pagopa/io-app
# feat: [IOPLT-360] Upgrades react-native-webview to latest version

> [!caution]
> This PR depends on #5506

## Short description
This PR aims to bump `react-native-webview` version to 13.8.1

## List of changes proposed in this pull request
- update library
- updates secrity patch to work with latest version of the library code

## How to test
Check the Webviews used through the app to check the consistency of the update

As a further test check the removal of the workaround introduced in #4460 doesn't introduce back the bug that should've been fixed in the base library
